geoh5py/shared/concatenation/concatenator.py

```diff
@@ -472,30 +473,40 @@ def remove_children(self, children: list | Concatenated):
 
             self.remove_entity(child)
 
-    def remove_entity(self, entity: Concatenated):
+    def remove_entity(self, entity: Concatenated | ConcatenatedPropertyGroup):
```

Comment:
In a drillholes group, can we remove an entity (a drillhole), but keep the similar properties on the other drillholes? Then we don't want to drop the property group, but just drop one property to the property group?

Unless each drillhole has a property group, and the name of the property group is recognize as a key by GA. (I can't remember the mechanics precisely).

Reply:
That's correct, property groups are unique on each drillhole. Mechanics are more restrictive than on regular objects since we can't have data not belonging to a prop_group. So if remove the group, we also need to delete the data underneath